### PR TITLE
Remove 'revamped website' notice on top #96

### LINKED ISSUE #96: Remove 'revamped website' notice on top
This can be removed. However, we should still mention where to report bugs.
image

### PR DESCRIPTION
**Pull request by RyanAlexanderH**
Fixes #96 
Here is the [preview](http://ryanalexanderh.github.io/website/)

### CODE REVIEW

File: contents/headings.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </head>
     <body leftmargin="5" topmargin="0">
         <span class="highlighted">
-          This is the revamped version of the module website. Report bugs/suggestions 
+          Report bugs/suggestions 
           <a href="https://github.com/nus-cs2103/website" target="_blank">at our issue tracker</a>.
         </span>
```
Comment (damithc) [inline contents/headings.html]: Lets remove the entire `<span>` and add a 'Feedback' to the top navigation menu.

**Comment by RyanAlexanderH** [comment]
Is it something like 'Feedback' on the top navigation menu linked to https://github.com/nus-cs2103/website?

**Comment by damithc** [comment]
Yes, but you can link to the [issue tracker](https://github.com/nus-cs2103/website/issues) page instead.

**Comment by wkurniawan07** [comment]
It seems like GitHub doesn't allow itself to be opened in a frame. In that case, open the link in a new tab instead (using `target="_blank"`).

**Comment by RyanAlexanderH** [comment]
Updated the link to the [new one](https://github.com/nus-cs2103/website/issues) and added the new tab too

**Comment by RyanAlexanderH** [comment]
Updated the link and now it should open the link in a new tab.

**Comment by wkurniawan07** [comment]
Ok this looks good :+1:

**Pull request merged**